packages/contentstack-auth/env.example

```diff
@@ -0,0 +1,8 @@
+BRANCH_ENABLED_API_KEY
+BRANCH_ENABLED_MGMT_TOKEN
+BRANCH_DISABLED_API_KEY
+BRANCH_DISABLED_MGMT_TOKEN
+BRANCH_ENABLED_DELIVERY_TOKEN
+BRANCH_DISABLED_DELIVERY_TOKEN
+BRANCH_ENABLED_ENVIRONMENT
+BRANCH_DISABLED_ENVIRONMENT
```

Comment:
@akashbhosale97 Looks like your test case utilising the original config. Instead of using the original config try setting the few keys in env and create your own config files after all test execution just clean up the config that is created for the test cases.

Env Keys that can create you new file. Check the config handler for more details.
```
CONFIG_NAME
ENC_CONFIG_NAME
```

For the cleanup script please refer [this](https://github.com/contentstack/cli/blob/tests/tokens-unit-tests/packages/contentstack-export/test/integration/clean-up.test.js#L29-L60)